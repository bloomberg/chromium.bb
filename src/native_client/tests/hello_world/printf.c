/*
 * Copyright 2009 The Native Client Authors. All rights reserved.
 * Use of this source code is governed by a BSD-style license that can be
 * found in the LICENSE file.
 */

/*
   It can't get much simpler than this (uh, except for noop.c).
*/

#include <stdio.h>
#include <stdlib.h>

int main(int argc, char* argv[]) {
  int i;

  for (i = 1; i < argc; ++i) {
    long x;
    printf("%s\n", argv[i]);

    x = strtol(argv[i], 0, 0);
    printf("%ld\n", x);
    printf("%lx\n", x);
    printf("0x%08lx\n", x);
    printf("\n");
  }
  return 0;
}
