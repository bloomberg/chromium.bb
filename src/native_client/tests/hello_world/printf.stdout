1000
1000
3e8
0x000003e8

1
1
1
0x00000001

0
0
0
0x00000000

